### DISCUSSION TITLE
Maximum Remote Control Sessions is 3... how to increase?

### DISCUSSION THREAD

**Discussion by PCProsCA** [immense/Remotely#572] [Q&A]
I sometimes run into the following error when trying to start a remote control session:
`Max number of concurrent sessions reached`

I found an item under server config for "Remote Control Session Limit" but I can't seem to change anything in there (i.e. I make a change and hit "Save" but nothing actually changes) even though I'm logged in as admin.

How can I increase the number of concurrent sessions?

Cheers,
Ryan
<img width="687" alt="2023-01-22 16_58_35-Remotely and 4 more pages - Work - Microsoft​ Edge" src="https://user-images.githubusercontent.com/123333458/213942678-8b54d3d5-4baf-43a5-a7aa-efdefa6c986f.png">
<img width="483" alt="2023-01-22 17_01_21-Remotely and 4 more pages - Work - Microsoft​ Edge" src="https://user-images.githubusercontent.com/123333458/213942679-08fc64af-5622-43cd-a7d0-77b8ab7a7085.png">

**Comment by PCProsCA** [comment]
ping, anyone have any idea here?  Seems like there should be a relatively simple way to fix this...

**Reply by bitbound** [reply]
It's a configurable option in `appsettings.json` (or in the Server Config page).

**Comment by bitbound** [comment]
Also worth noting.  I encountered some bugs where remote control processes aren't getting stopped after the session ends.  This will be addressed in the next release.
